content/docs/install.md

```diff
@@ -1,30 +1,22 @@
 # Installing CML as a Package
 
-CML comes pre-installed in our provided
-[Docker Images](/doc/self-hosted-runners#docker-images). Alternatively, GitHub
-users can also use the [`setup-cml` action](/doc/start/github#setup-action).
+<admon>
 
-However, in all other cases, CML can be installed directly as a Node.js package
-using the package manager `npm` ([see below](#installing-nodejs)):
+Installing CML directly in CI environment is not typically needed, as comes
+pre-installed in our provided [Docker Images]. Alternatively, GitHub users can
+use the [`setup-cml` action].
```

Comment:
Added an admon to clarify that you probably don't want to do this.

Reply:
Are the changes to this file (and nav) good at least? If you don't want to remove the confusing menu entry.

Otherwise feel free to take this over, close it, etc. Thanks @iterative/cml